install_builder/deadline-cloud-for-keyshot.xml

```diff
@@ -66,28 +66,21 @@
                     </conditionRuleList>
                     <actionList>
                         <!-- KeyShot 2023 env var -->
-                        <setInstallerVariable name="keyshot_resources_directory" value="${env(KEYSHOT12)}"/>
+                        <setInstallerVariable name="keyshot_scripts_directory_default" value="${env(KEYSHOT12)}\Scripts"/>
```

Comment:
Should this be `keyshot_scripts_directory` instead of `keyshot_scripts_directory_default`?

Reply:
Eek, yeah. Rebase issue.